src/cloudant/index.py

```diff
@@ -0,0 +1,173 @@
+#!/usr/bin/env python
+"""
+_utils_
+
+General utilities
+
+
+"""
+import json
+
+from collections import Sequence
+
+ALL_ARGS = [
+    "descending",
+    "endkey",
+    "endkey_docid",
+    "group",
+    "group_level",
+    "include_docs",
+    "inclusive_end",
+    "key",
+    "limit",
+    "reduce",
+    "skip",
+    "stale",
+    "startkey",
+    "startkey_docid"
+]
+
+BOOLEAN_ARGS = [
+    'include_docs',
+    'inclusive_end',
+    'reduce',
+    'group',
+    'descending'
+]
+
+ARRAY_ARGS = [
+    'startkey',
+    'endkey'
+]
+STRING_ARGS = [
+    'key',
+]
+
+
+def python_to_couch(kwargs):
+    """
+    _python_to_couch_
+
+    Translator method to flip python style
+    options into couch query options, eg True => 'true'
+    """
+    for b in BOOLEAN_ARGS:
+        if b in kwargs:
+            if kwargs[b]:
```

Comment:
The following will evaluate to true: `"banana", 5, object(), some_function` -- check bool before converting? Same for false.